README.md

```diff
@@ -231,6 +231,41 @@ The full development and release path now is:
 
 <!--tutorial end-->
 
+### GitHub Actions
+
+You can use `bump-my-version` as part of a GHA workflow. Here is an example of a workflow that bumps the version, commits the change, and tags the commit.
+
+```yaml title="GitHub Actions Workflow"
+name: Bump version
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+
+      - name: Bump version
+        uses: callowayproject/bump-my-version@master
+        env:
+          BUMPVERSION_TAG: "true"
+        with:
+          args: "major"
```

Comment:
Instead of hardcoding this in the example, you might want to add an `args` input to the `workflow_dispatch` [trigger](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#providing-inputs). This would make it more flexible.

Reply:
Do you mean to add the parameter ot the parameter to the example workflow itself, right? Because parametrized workflow inputs aren't available when creating just an action definition. Unfortunately.

Reply:
Yes, even if you did something like (on line 241)

```yaml
on:
  workflow_dispatch:
    inputs:
      bump_type:
        description: 'Bump type'
        required: true
        default: 'patch'
        type: choice
        options:
        - major
        - minor
        - patch
```

and then on 255:

```yaml
        with:
           args: ${{ inputs.bump_type }}
```

Reply:
Added, along with an example of how it could be possible to further utilize the bump step's outputs.